lib/FinderNavigation/FinderItem.js

```diff
@@ -8,13 +8,20 @@ const FinderItem = ({
   hoverSettings,
   selected,
   children,
+  noIndentation,
   ...rest
 }) => {
-  const classNames = cx(`finder__item ${className}`, {
-    'is-active': active,
-    'finder__item--hover-settings': hoverSettings,
-    'finder__item--selected': selected
-  });
+  const classNames = cx(
+    `${
+      noIndentation ? 'finder__item-no-indentation' : 'finder__item'
```

Comment:
Following BEM what normally happens is `finder__item` sets a base and then you apply modifiers on top of that. Ideally `finder__item` would always be set and then you would add `finder__item-no-indentation` on top like so;

```
cx('finder__item', {
  noIndentation: 'finder__item-no-indentation;
})
```

Was there a reason why we didn't do that for this case?